ark/jsonschema/composition.ts

```diff
@@ -0,0 +1,60 @@
+import { printable } from "@ark/util"
+import { type, type JsonSchema, type Type } from "arktype"
+import { parseJsonSchema } from "./json.ts"
+
+const validateAllOfJsonSchemas = (jsonSchemas: readonly JsonSchema[]): Type =>
```

Comment:
Should this be called something like `parseAllOf` or similar? I'd expect validate to be checking data

Reply:
Renamed to `parseAllOfJsonSchema` within a4defa88